packages/static/grub-config/files/grub.cfg

```diff
@@ -59,6 +59,10 @@ insmod squash4
 insmod serial
 insmod regexp
 loadfont unicode
+if [ "${grub_platform}" = "efi" ]; then
+    ## workaround for grub2-efi bug: https://bugs.launchpad.net/ubuntu/+source/grub2/+bug/1851311
+    rmmod tpm
```

Comment:
does that mean that we cannot use tpm encryption with EFI atm?

Reply:
This is only for measurement of the artifacts into the tpm registers. Because for now we only support measured boot under uki which uses systemd-boot this should not affect us. 

Let's hope that this is fixed if we ever want to use those measurements for something with grub 🤣

Btw this only affects debian derivatives. I hope this is fixed once debian bumps to the 2.12 version

Reply:
gotcha, thanks for clarifying